gui/slick/views/home.mako

```diff
@@ -41,353 +41,19 @@
-                        return (name || '');
-                    % endif
-                },
-                network: '[data-network]',
-                date: function( itemElem ) {
-                    var date = $( itemElem ).attr('data-date');
-                    return date.length && parseInt( date, 10 ) || Number.POSITIVE_INFINITY;
-                },
-                progress: function( itemElem ) {
-                    var progress = $( itemElem ).attr('data-progress');
-                    return progress.length && parseInt( progress, 10 ) || Number.NEGATIVE_INFINITY;
-                }
-            }
-        });
-    });
-
-    $('#postersort').on( 'change', function() {
-        var sortValue = this.value;
-        $('#container').isotope({ sortBy: sortValue });
-        $('#container-anime').isotope({ sortBy: sortValue });
-        $.get(this.options[this.selectedIndex].getAttribute('data-sort'));
-    });
-
-    $('#postersortdirection').on( 'change', function() {
-        var sortDirection = this.value;
-        sortDirection = sortDirection == 'true';
-        $('#container').isotope({ sortAscending: sortDirection });
-        $('#container-anime').isotope({ sortAscending: sortDirection });
-        $.get(this.options[this.selectedIndex].getAttribute('data-sort'));
-    });
-
-    $('#popover')
-        .popover({
-          placement: 'bottom',
-          html: true, // required if content has HTML
-          content: '<div id="popover-target"></div>'
-        })
-        // bootstrap popover event triggered when the popover opens
-        .on('shown.bs.popover', function () {
-          // call this function to copy the column selection code into the popover
-          $.tablesorter.columnSelector.attachTo( $('#showListTableShows'), '#popover-target');
-          % if sickbeard.ANIME_SPLIT_HOME:
-          $.tablesorter.columnSelector.attachTo( $('#showListTableAnime'), '#popover-target');
-          % endif
-        });
-});
-</script>
+<meta data-var="sickbeard.SORT_ARTICLE" data-content="${sickbeard.SORT_ARTICLE}">
+<meta data-var="sickbeard.FILTER_ROW" data-content="${sickbeard.FILTER_ROW}">
+<meta data-var="sickbeard.ANIME_SPLIT_HOME" data-content="${sickbeard.ANIME_SPLIT_HOME}">
+<meta data-var="sickbeard.POSTER_SORTBY" data-content="${sickbeard.POSTER_SORTBY}">
+<meta data-var="sickbeard.POSTER_SORTDIR" data-content="${sickbeard.POSTER_SORTDIR}">
+<meta data-var="sickbeard.FUZZY_DATING" data-content="${sickbeard.FUZZY_DATING}">
+<meta data-var="sickbeard.DATE_PRESET" data-content="${sickbeard.DATE_PRESET}">
+<meta data-var="sickbeard.TIME_PRESET" data-content="${sickbeard.TIME_PRESET}">
+<meta data-var="sickbeard.TRIM_ZERO" data-content="${sickbeard.TRIM_ZERO}">
+<meta data-var="max_download_count" data-content="${max_download_count}">
+<meta data-var="layout" data-content="${layout}">
+<meta data-var="fuzzydate" data-content="${fuzzydate}">
+<script type="text/javascript" src="${sbRoot}/js/new/home.js"></script>
```

Comment:
For those wondering, for now we're using meta tags to store information we need in js until the v2 api gets view paths added.